framework/Makefile

```diff
@@ -46,6 +46,11 @@ ifeq ($(CONFIG_WIFI_MANAGER), y)
 include src$(DELIM)wifi_manager$(DELIM)Make.defs
 endif
 
+ifeq ($(CONFIG_ST_THINGS), y)
+CFLAGS += -I$(TOPDIR)$(DELIM)..$(DELIM)framework$(DELIM)src$(DELIM)wifi_manager
```

Comment:
Why do you include this folder?

Reply:
we can remove this include folder after wifi_manager modify.